### ISSUE TITLE
2 Instances and freeze

### ISSUE THREAD

**Issue by WeePhix** [Iskallia/Vault-public-S1#307]
**Bug description**
I downloaded Vault Hunters from CurseForge and ran the latest version. When I clicked "PLAY", 2 Minecraft instances opened. 1 is vanilla Minecraft and 1 is probably Vault Hunters. The vanilla instance loads with normal speed, the modded instance however doesn't load and gets stuck at a random point while loading the mods. If I close the vanilla instance, the modded instance quits too.

**To Reproduce**
1. Install and run Vault Hunters 1.2
2. Press "PLAY" in the Minecraft launcher
3. Wait

**Expected behavior**
Only a modded instance is supposed to run and load.

**Screenshots or Crash Report**
![image](https://user-images.githubusercontent.com/83700122/117123753-b8618d80-ad97-11eb-886f-874de298a87e.png)

**Info (please complete the following information):**
- World Type: None
- The_Vault Version [e.g. 1.0.1]: 1.0.2

**Additional context**
[latest.log](https://github.com/Iskallia/Vault-public/files/6426220/latest.log)
Please help

**Comment by BONNe** [comment]
Did you launch it from corseforge?

**Comment by WeePhix** [comment]
> Did you launch it from curseforge launcher?

Yes, I did. I also ran it from there

**Comment by BONNe** [comment]
Hmm, it looks like issue with curseforge launcher.

**Comment by WeePhix** [comment]
> Hmm, it looks like issue with curseforge launcher.

But other modpacks work fine. Or do you mean something else?

**Comment by BONNe** [comment]
Hmm, I just downloaded a fresh version via CurseForge and I did not run into any issues with starting the pack.

Did you changed something in run parameters?

**Comment by WeePhix** [comment]
> Hmm, I just downloaded a fresh version via CurseForge and I did not run into any issues with starting the pack.
> 
> Did you changed something in run parameters?

No I left everything as it was. It might be my PC acting up...

**Comment by WeePhix** [comment]
SOLUTION: I uninstalled and reinstalled the Curse launcher and it works fine after that. Make sure to delete the modpack folders too.

**Comment by waidesoper** [comment]
So this issue can be closed?

**Comment by WeePhix** [comment]
> So this issue can be closed?

Yes. Im new to github and dont know if/how i can close it... it was on my side
